getAllFeatureFlags/main.go

```diff
@@ -43,28 +43,24 @@ func getAllFeatureFlags(db *dynamodb.DynamoDB) ([]utils.FeatureFlagResponse, err
 func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
 	db := database.CreateDynamoDB()
 
-	log.Println(db, " is the database")
 	featureFlagsResponse, err := getAllFeatureFlags(db)
 
 	if err != nil {
 		return utils.ServerError(err)
 	}
 
 	if featureFlagsResponse == nil {
-		return events.APIGatewayProxyResponse{
-			Body: "No entries found.",
-			StatusCode : http.StatusNotFound,
-		}, nil
+		utils.ClientError(http.StatusNotFound, "No feature flags found :( ")
```

Comment:
Please add `return` here

Reply:
Also I think the error message `No feature flags found` is better

Reply:
Sure.

Reply:
done resolving this